### PR TITLE
test/parsec: minimally cover add/remove/opaque after DKG

### LINKED ISSUE #333: Extend test to cover Parsec votes when non voters DKG participants are present
Follow #327 design task.

Validate Parsec behavior when non voters participate in the gossip graph.
At the minimum, validate Add,Remove,and opaque payload.

### PR DESCRIPTION
**Pull request by jeanphilippeD**
Closes #333 
Also use consensus mode single as routing does.

### CODE REVIEW

File: tests/integration_tests.rs [hunk 1]
```diff
@@ -279,71 +279,95 @@ fn run_split_dkg() {
...
+    let mut env = Environment::with_consensus_mode(SEED, ConsensusMode::Single);
+    let named_peer_ids = PeerId::named_peer_ids();
+
+    let final_peer_ids: BTreeSet<_> = named_peer_ids[1..5].iter().cloned().collect();
+    let genesis: BTreeSet<_> = named_peer_ids[0..4].iter().cloned().collect();
+    let single_add_remove: BTreeSet<_> = named_peer_ids[1..5].iter().cloned().collect();
+
+    let dkgs = vec![(single_add_remove, "single_add_remove".to_string())];
+    let additional_events = vec![
+        (500, ObservationEvent::AddPeer(named_peer_ids[4].clone())),
```
Comment (fizyk20) [inline tests/integration_tests.rs]: Would be cool to make sure that these are only voted on after the DKG actually finishes - but I understand that this part is out of scope of this PR and that's why there is "minimally" in its title. Not a blocker, anyway.

Comment (jeanphilippeD) [reply tests/integration_tests.rs]: Well ideally we would make sure the test can cover both the case when DKG finished, or is in progress, as both seem relevant. but it is a bit difficult to control that this will happen.
